pkg/controller/datavolume-controller.go

```diff
@@ -367,6 +380,32 @@ func (c *DataVolumeController) syncHandler(key string) error {
 	return nil
 }
 
+func (c *DataVolumeController) scheduleProgressUpdate(dataVolumeName, dataVolumeNamespace string, pvcUID types.UID) {
+	for {
+		time.Sleep(2 * time.Second)
+		dataVolume, err := c.cdiClientSet.Cdi().DataVolumes(dataVolumeNamespace).Get(dataVolumeName, metav1.GetOptions{})
```

Comment:
Should we maybe use the dataVolumesLister here instead?  So we're not always going to the API server?

Reply:
yeah good plan.